css-flexbox-abspos/index.html

```diff
@@ -0,0 +1,32 @@
+---
+feature_name: New CSS Flexbox behavior for absolute-positioned children
+chrome_version: 52
+feature_id: 6600926009753600
+---
+
+<h3>Background</h3>
+<p>A previous version of the CSS Flexible Box Layout specification set the static position of absolute-positioned children as though they were a 0x0 flex item. The latest version of the spec takes them fully out of flow and sets the static position based on align and justify properties.</p>
+
+<p>The sample CSS and HTML below demonstrate this. Starting with Chrome 52 or later, the green box will be perfectly centered in the red box. In other browsers, the top left corner of the green box will be in the top center of the red box.</p>
```

Comment:
For the "In other browsers..." bit, is it the case that no other browsers behave the way Chrome 52 does? Maybe just say "In earlier versions of Chrome..." without making a statement about what other browsers have implemented, especially since other browsers might add support in the future.